domaintools_misp/base.py

```diff
@@ -315,12 +315,12 @@ def extract_nested_value(self, value, label):
                     "tags": ["DomainTools", "Guided Pivot"],
                 }
             )
-        elif "count" not in value:
+        elif value and "count" not in value:
```

Comment:
Can we do the checking of truthness of `value` before the if statements? As this condition only checks if it goes to `elif`.

Reply:
Addressed already

Reply:
Awesome! I think we can remove the checking of `value` in elif